### PR TITLE
feat(files): Added Trial Chamber Panorama

### LINKED ISSUE #416: [GUI/Panorama] Trial Chamber Panorama
> Replaces the default Minecraft menu panorama with a 360° view of a Trial Chamber! (from 1.21)

Pack Parity

### PR DESCRIPTION
**Pull request by NSPC911**
Added Trial Chamber Panorama
If you (`@drav0011`) wants, I can also add Pale Garden Panorama

<details>
  <summary>Issues Resolved:</summary>
  <ul>
    <li>Resolves #416</li>
  <ul>
</details>

By checking the following boxes with an X, you ensure that:

- [x] The pack was tested ingame in at least one device.
- [x] The pack is an existing BT pack, is a missing pack from VT or is an accepted pack/change in a discussion.
- [x] The pack code follows the style guide.
- [x] The commits follow the contribution guidelines.
- [x] The PR follows the contribution guidelines.

- [ ] (Optional) Tested in Windows
- [x] (Optional) Tested in Android
- [ ] (Optional) Tested in iOS
- [ ] (Optional) Tested in any console
- [ ] (Optional) Tested in BDS

### CODE REVIEW

File: resource_packs/packs.json
Comment (SquatchHunter) [inline resource_packs/packs.json:1]: Vanilla Tweaks now include a "version from" in their descriptions
"description": "Replaces the default Minecraft menu panorama with a 360° view of a Trial Chamber! (from 1.21)",

**Review by SquatchHunter — CHANGES_REQUESTED** [review]

File: resource_packs/packs.json
Comment (NSPC911) [reply resource_packs/packs.json:1]: specify sub version like 1.21`.40` or jist 1.21?

Comment (SquatchHunter) [reply resource_packs/packs.json:1]: they dont have stubs so I wouldn't - probably a personal preference / DrAv question.

**Comment by SquatchHunter** [comment]
this also resolves: #416

**Review by SquatchHunter — APPROVED** [review]
Lgtm

**Review by PAPISOP — APPROVED** [review]
lgtm

File: resource_packs/packs.json
Comment (drav0011) [reply resource_packs/packs.json:1]: I would add subversion for the panoramas which are from a subversion
Example 
<Update 1.21 panorama idk what it was> (1.21)
<Update 1.21.40 panorama pale oak and stuff> (1.21.40)

And talking about pale oak I would add once the next update releases no point now as is the default one
Can add a draft pr if you want and keep it there

**Review by drav0011 — APPROVED** [review]
lgtm

**Pull request merged**